.env.example

```diff
@@ -1,5 +1,8 @@
 CHECKR_API_URL='https://api.checkr-staging.com'
 REACT_APP_CHECKR_PARTNER_URL='https://partners.checkrhq-staging.net'
+
+# Replace the following values with the client_id and client_secret of your partner application
```

Comment:
Could you add a link to that page.

https://dashboard.checkrhq-staging.net/account/applications